src/apply-preserving-inline-style.js

```diff
@@ -14,6 +14,23 @@
 
 (function(scope, testing) {
 
+  /**
+   * IE/Edge do not support `transform` styles for SVG elements. Instead,
+   * `transform` attribute can be animated with some restrictions.
+   * See https://connect.microsoft.com/IE/feedback/details/811744/ie11-bug-with-implementation-of-css-transforms-in-svg,
+   * https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/1173754/,
+   * https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/101242/, etc.
+   * The same problem is exhibited by pre-Chrome Android browsers (ICS).
+   * Unfortunately, there's no easy way to feature-detect it.
+   */
+  function updateSvgTransformAttr(window) {
+    if (window._webAnimationsUpdateSvgTransformAttr == null) {
```

Comment:
It appears that we don't. We don't obfuscate the tests so this wouldn't work anyway if we did.